kustomize/move2kube/base/03-knative-resources.yaml

```diff
@@ -0,0 +1,128 @@
+
+    ref:
+      apiVersion: v1
+      kind: Service
+      name: workflow_name
+---
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: transformation-saved-trigger-move2kube
+spec:
+  broker: brokerName
+  filter:
+    attributes:
+      type: transformation_saved
+  subscriber:
+    ref:
+      apiVersion: v1
+      kind: Service
+      name: workflow_name
+---
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  name: brokerName
+---
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: kfunction_name
+spec:
+  template:
+    metadata:
+      name: kfunction_name-v1
+    spec:
+      initContainers:
+        - name: volume-mount-hack
+          image: quay.io/quay/busybox:latest 
+          securityContext:
+            runAsUser: 0
+            allowPrivilegeEscalation: true
+          command: [ "sh", "-c", "cp /tmp/.ssh/id_rsa /etc/pre-install/. && chown 185 /etc/pre-install/id_rsa" ]
+          volumeMounts:
+            - name: ssh-priv-key
+              mountPath: "/tmp/.ssh/id_rsa"
+              subPath: id_rsa
+              readOnly: true
+            - name: pre-install
+              mountPath: /etc/pre-install
+      containers:
+        - image: serverless-workflow-m2k-kfunc
+          imagePullPolicy: Always
+          env:
+            - name: EXPORTED_FUNC
+              value: saveTransformation
+            - name: SSH_PRIV_KEY_PATH
+              value: /home/jboss/.ssh/id_rsa
+          envFrom:
+          - configMapRef:
+              name: move2kube-config # defines BROKER_URL and MOVE2KUBE_API         
+          name: user-container
+          volumeMounts:
+            - name: pre-install
+              readOnly: true
+              mountPath: "/home/jboss/.ssh/id_rsa"
+              subPath: id_rsa
+            - name: ssh-pub-key
+              readOnly: true
+              mountPath: "/home/jboss/.ssh/id_rsa.pub"
+              subPath: id_rsa.pub
+
+          readinessProbe:
+            successThreshold: 1
+            tcpSocket:
+              port: 0
+      volumes:
+        - name: ssh-priv-key
+          secret:
+            secretName: sshSecretName
+            defaultMode: 384
+        - name: ssh-pub-key
+          secret:
+            secretName: sshSecretName
+        - name: pre-install
+          emptyDir: { }
+
+---
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: m2k-save-transformation-event
+spec:
+  broker: brokerName
+  filter:
+    attributes:
+      type: save-transformation
+  subscriber:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: kfunction_name
+---
+apiVersion: sources.knative.dev/v1
+kind: SinkBinding
+metadata:
+  name: workflow_name-sb
+spec:
+  subject:
+    apiVersion: v1/apps
+    kind: Deployment
+    selector:
+      matchLabels:
+        app: workflow_name
+  sink:
+    ref:
+      apiVersion: eventing.knative.dev/v1
+      kind: Broker
+      name: brokerName
```

Comment:
this could be removed 
Maybe it will be usable once changes are made by richard (https://docs.google.com/document/d/1GrVKanrwsEr3r1VlepErHOnq8dVI4dr8EeQcecTvqBE/edit#heading=h.ds8q4xtkmu64) because currently, if we have a sinkbinding, it will inject the K_SINK into the deployment but the sonataflow operator will reconcile and remove it but then the sinkbinding will add it again...and so forth
But I'll add a K_SINK value in the env config map (see https://github.com/parodos-dev/serverless-workflows-helm/pull/44/files#diff-b7d055be1f6eb719d57e746f84b5703723b6f734263479bff5ee0dee8dd58d2fR259)